rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -3044,6 +3047,11 @@ private Pair<BtcTransaction, List<UTXO>> createMigrationTransaction(Wallet origi
         logger.debug("[createMigrationTransaction] Balance to migrate: {}", expectedMigrationValue);
         for(;;) {
             BtcTransaction migrationBtcTx = new BtcTransaction(originWallet.getParams());
+            if (activations.isActive(ConsensusRule.RSKIP376)){
+                migrationBtcTx.setVersion(BRIDGE_BTC_TX_VERSION_2);
+            } else {
+                migrationBtcTx.setVersion(BRIDGE_BTC_TX_LEGACY_VERSION);
+            }
```

Comment:
```suggestion
            if (activations.isActive(ConsensusRule.RSKIP376)){
                migrationBtcTx.setVersion(2);
            }
```
I think that covers it, it's version 1 by default

Reply:
I agree, but it does not hurt to have an assertion of version 1.  There is a possibility of bitcoinj-thin implementation being modified and introducing a bug by mistake.